composer.json

```diff
@@ -1,20 +1,27 @@
 {
-    "name": "viison/address-splitter",
+    "name": "pickware/address-splitter",
     "description": "Tries to split an address line into street name, house number and other additional information like building, apartment information etc.",
     "license": "MIT",
     "authors": [
         {
-            "name": "VIISON GmbH",
-            "email": "contact@viison.com"
+            "name": "Pickware GmbH",
+            "email": "support@pickware.de"
         }
     ],
     "require": {
-        "php": ">=5.3.2"
+        "php": ">=5.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.0"
+        "phpunit/phpunit": "^5.7"
```

Comment:
If you go to the trouble of fixing the code style, we should add the phpcs config from https://github.com/pickware/style-guide here to make sure it stays this way.

Reply:
We'd better put that off until later. I can create an issue for it.